src/include/kdbprivate.h

```diff
@@ -518,10 +518,13 @@ int elektraValidateKeyName(const char *name, size_t size);
  */
 #define ELEKTRA_MAX_ARRAY_SIZE (21)
 
+#if !defined(__cplusplus)
```

Comment:
This was the case some time ago (because mingw was not able to build them because of int64_t). But now they should work everywhere.

In kdbprivate.h there was still a int64_t, I fixed this in 9734b95067cfb3b588cc7352b464e730b891e5cc